browser/ui/views/toolbar/brave_vpn_status_label.cc

```diff
@@ -0,0 +1,54 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/views/toolbar/brave_vpn_status_label.h"
+
+#include "brave/browser/brave_vpn/brave_vpn_service_factory.h"
+#include "brave/browser/themes/theme_properties.h"
+#include "brave/grit/brave_generated_resources.h"
+#include "chrome/browser/ui/browser.h"
+#include "chrome/browser/ui/views/frame/browser_view.h"
+#include "ui/base/l10n/l10n_util.h"
+#include "ui/base/theme_provider.h"
+
+BraveVPNStatusLabel::BraveVPNStatusLabel(Browser* browser)
+    : browser_(browser),
+      service_(BraveVpnServiceFactory::GetForProfile(browser_->profile())) {
+  observation_.Observe(service_);
+  SetAutoColorReadabilityEnabled(false);
+  UpdateState();
+
+  if (auto* provider =
+          BrowserView::GetBrowserViewForBrowser(browser_)->GetThemeProvider()) {
```

Comment:
What about `OnThemeChanged()`

Reply:
At first I tried to overriding that method but I realized that this menu widget doesn't have theme provider.
Because of that, I got theme color from browser window not its root widget.
And I think setting color in ctor seems fine in this case because menu widget is destroyed whenever it lost focus.
It's very difficult to live while user change theme.

Reply:
i see. thanks for the explanation. marking as resolved.